src/main/java/SpringJPA/WebSecurityConfig.java

```diff
@@ -42,14 +42,17 @@ protected void configure(AuthenticationManagerBuilder auth) throws Exception {
     protected void configure(HttpSecurity http) throws Exception {
         http
                 .authorizeRequests()
-                    .antMatchers("/").permitAll()
+                    .antMatchers("/","/user/makeAPICall").permitAll()
                     .anyRequest().authenticated()
                     .and()
                 .formLogin()
                     .permitAll()
                     .and()
+                .csrf()
+                .disable()
```

Comment:
Can you find a way to do this without disabling csrf?

Reply:
Addressed your issue.